vcs/displayplot.py

```diff
@@ -25,14 +25,46 @@
 #
 from . import VCS_validation_functions
 import vcs
+import tempfile
 from .xmldocs import listdoc  # noqa
+from functools import partial
+
+try:
+    import IPython.display
+    import ipywidgets
```

Comment:
My naive thought would be you set some variable here which you can later use to determine whether you have widgets or not.  Otherwise, do you just assume the import worked later on?